observations/obs_converters/NCEP/prep_bufr/prep_bufr.rst

```diff
@@ -1,67 +1,315 @@
+   water vapor (246), sat winds (252,253,255), and ship obs (280, 281,
+   282). Additionally, it will include observations with specified qc
+   types only. Skip to the prepfbufr **Namelist** section at the bottom
+   of this page for more available namelist controls.
+
+-  Within the
+   ``$DART_DIR/observations/obs_converters/NCEP/prep_bufr/work``
+   directory, edit the ``prepbufr.csh`` file and change *BUFR_dir*,
+   *BUFR_idir*, *BUFR_odir*, and *BUFR_in* to match the locations and
+   format of the data you downloaded. A little trial and error might be
+   necessary to get these set correctly. For example:
+  
+   ::
+
+      set BUFR_dir  = ../data
+      set BUFR_idir = ${BUFR_dir}/prepqm
+      set BUFR_idir = ${BUFR_dir}/prepout
+      ..
+      set BUFR_in = ${BUFR_idir}/prepqm${sdtg}
+      
+
+-  Copy over the executables from ``../exe``, and run the *prepbufr.csh*
+   script for a single day at a time:
+
+   ::
+
+      cd $DART_DIR/observations/obs_converters/NCEP/prep_bufr/work
+      cp ../exe/\*.x .
+      ./prepbufr.csh 2017 04 27
+
+
+
+-  Your PREPBUFR files have now been converted to an intermediate ASCII
+   format. Confirm that ``temp_obs.20170427*`` files within your 
+   ``~/data/prepout directory`` exist. Please note that the script can function
+   with only the *06* prepqm input file, but will need the
+   *12* and *18* files to run to completion. 
+
+
+-  There is another observation converter to take the
+   observations from this format and write them into the native DART
+   format. Edit the *input.nml* namelist file in the
+   *DART_DIR/observations/obs_converters/NCEP/ascii_to_obs/work*
+   directory. For this example:
+
+   ::
+
+      &ncepobs_nml
+         year       = 2017,
+         month      = 4,
+         day        = 27,
+         tot_days   = 3,
+         max_num    = 800000,
+         select_obs = 0,
+         ObsBase = '../../prepbufr/data/prepout/temp_obs.',
+         daily_file = .false.,
+         lat1       = 15.0,
+         lat2       = 60.0,
+         lon1       = 200.0,
+         lon2       = 330.0
+         /
+
+   Choosing "select_obs = 0" will select all the observations in the
+   ASCII file. Set "ObsBase" to the directory you output the files from
+   during the last step. If you wish to choose specific observations
+   from the ASCII intermediate file or control other program behavior,
+   there are many namelist options documented on the
+   `create_real_obs <../../../observations/obs_converters/NCEP/ascii_to_obs/create_real_obs.html>`__
+   page.
+
+-  It is now time to build *ascii_to_obs* programs. Run the following:
+
+   ::
+
+      cd $DART_DIR/observations/obs_converters/NCEP/ascii_to_obs/work
+      ./quickbuild.sh
+
+-  Run the *create_real_obs* program to create the DART observation
+   sequence files:
+
+
+   ::
+
+      cd $DART_DIR/observations/obs_converters/NCEP/ascii_to_obs/work
+      ./create_real_obs
```

Comment:
I ran this and it created the obs sequence files for 06, 12, and 18Z. But it also gave me the following. Is it something we should look into? Or did I do something wrong?  

```
 ERROR FROM:
  source : utilities_mod.f90
  routine: open_file:
  message:  Cannot open file "../../prep_bufr/data/prepout/temp_obs.2017062724" for read
  message: ...  File may not exist or permissions may prevent the requested operation
  message: ...  Error code was           29
  ```

Reply:
hi moha - i think this may be related to the issue that observations directly on the boundary time between 6H windows can occur in either the previous or following file.  i think the converter tries to open the next file to collect any obs which are on the initial time boundary.

Reply:
i just went through the prepbufr script again.  if you say "daily=true" it makes a single 24 hour output file.  if you say "daily=false" it still tries to convert a full day's worth of data but outputs 4 6-hour files.  to finish up a day, it needs the 0Z and 6Z files from the following day to be available.  (it creates the 6, 12, and 18 h files just fine, but ends with an error when it tries to open the 0Z and 6Z file from the next day.)